src/briefcase/commands/new.py

```diff
@@ -378,6 +378,7 @@ def select_option(
         default: str | None,
         options: Sequence[str],
         override_value: str | None,
+        external_hooks: str | None = None,
```

Comment:
This doesn't need to be a configurable argument. An external hook is only needed for one specific template selection, so it would be better to put the extra content directly into the intro prompt.

Reply:
I pushed a commit to remove this argument and put it directly into the intro. Let me know what you think of the new version